csrankings-e.csv

```diff
@@ -683,6 +683,7 @@ Eugenio Zimeo,University of Sannio,https://www.unisannio.it/en/users/Eugenio-Zim
 Eui-Nam Huh,Kyung Hee University,http://icnslab.net/#/introduction,0ATpZl0AAAAJ
 Eui-nam Huh,Kyung Hee University,http://icnslab.net/#/introduction,0ATpZl0AAAAJ
 Euijong Whang,KAIST,http://stevenwhang.com,w6hts30AAAAJ
+Euiseong Seo,Sungkyunkwan University,http://csl.skku.edu,VjycY14AAAAJ

```

Comment:
Not clearly in CS; please clarify.

Reply:
Here is the profile of Euiseong Seo: http://csl.skku.edu/People/Euiseong.
As he stated in the page, he is affiliated with the College of Computing and Informatics at SKKU.
The faculty list of the Dept. of CSE also lists his name as a professor: https://cse.skku.edu/eng_cse/faculty.do
Search for "Eui Seong" (with an empty space between "Eui" and "Seong").